### ISSUE TITLE
case sensitivity inconsistent, YES ignored

### ISSUE THREAD

**Issue by jlittlenz** [openSUSE/snapper#631]
snapper set-config number_cleanup=yes
    Invalid configdata.
but

    snapper set-config NUMBER_CLEANUP=YES

is silently accepted, and means "no".  Please, either accept YES as yes, or give an error.

**Comment by aschnell** [comment]
Yes, only the key is checked. In general the value can be lowercase, e.g. for the SUBVOLUME.
